### ISSUE TITLE
Braker3 or Braker1/Braker2/TSEBRA?

### ISSUE THREAD

**Issue by JohnUrban** [Gaius-Augustus/BRAKER#576]
Hi,

Thanks for the great tool.

I am gearing up to do an annotation with both RNA-seq and protein data.

Will you be recommending Braker3 moving forward for using both RNA-seq and protein?

Or are you still recommending the combination of Braker1/Braker2/TSEBRA?

Best,

John

p.s. Is there any merit in running Braker1, Braker2, and Braker3, then using TSEBRA on all 3?

**Comment by vpymerel** [comment]
Hi, 

I would like to add a question to this,

Why Braker**1**/Braker2/TSEBRA and not Braker**2**/Braker2/TSEBRA,

It is not clear for me,

Thank you,
V

**Comment by KatharinaHoff** [comment]
Merging to identical gene sets would not make much sense, that's why.

**Comment by vpymerel** [comment]
Of course, but in the use case of TSEBRA mentionned on github,  BRAKER1 is used with RNA evidence, and then BRAKER2 with protein evidence,

Why not using BRAKER2 with RNA evidence, then BRAKER2 with protein evidence ? Or is it was is actually done, and "Run BRAKER1 and BRAKER2 for example with ..." should be understood "Run a first run of BRAKER2 and a second run of BRAKER2" ?

**Comment by KatharinaHoff** [comment]
When we say BRAKER1, we mean braker.pl with RNA-Seq evidence. The team at
Georgia Tech prefers terminology like this: BRAKER1 is with RNA-Seq,
BRAKER2 is with proteins. It's all the same braker.pl script, just
different publications.

**Comment by vpymerel** [comment]
Thank you ! It make more sense for me now !

**Comment by skagawa2** [comment]
Sorry if I'm misunderstanding your answers here, but it doesn't seem like the original question was answered. Should we be using BRAKER3 or is BRAKER1+BRAKER2->TSEBRA recommended from now on?

**Comment by LarsGab** [comment]
Hi, 
we recommend using BRAKER3 over combining BRAKER1 and BRAKER2 with TSEBRA. 
Best, 
Lars

**Comment by skagawa2** [comment]
Thank you for the clarification!
